assets/css/v2/dup/normal_header.scss

```diff
@@ -85,3 +85,10 @@
   color: white;
   position: absolute;
 }
+
+.normal-header--yellow {
+  background-color: $alert-yellow;
+  .normal-header-title__text {
```

Comment:
I know this `title__text` bit probably needs to be down here so it isn't overridden above, but should `background_color` be moved up so all `background-color`s live in the same place?

Reply:
Yeah, that's probably good for the future!